### PR TITLE
Build aarch/ppc packages

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Not sure what's going on with the re-render ..
```
Traceback (most recent call last):
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_build/variants.py", line 519, in get_package_combined_spec
    validate_spec(spec)
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_build/variants.py", line 138, in validate_spec
    raise ValueError("Variant configuration errors: \n{}".format(errors))
ValueError: Variant configuration errors: 
["zip_key entry fortran_compiler_version in group ['c_compiler_version', 'cxx_compiler_version', 'fortran_compiler_version'] does not have any settings"]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/tobias/anaconda3/bin/conda-smithy", line 10, in <module>
    sys.exit(main())
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_smithy/cli.py", line 613, in main
    args.subcommand_func(args)
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_smithy/cli.py", line 419, in __call__
    self._call(args, tmpdir)
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_smithy/cli.py", line 430, in _call
    temporary_directory=temporary_directory,
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_smithy/configure_feedstock.py", line 1953, in main
    render_azure(env, config, forge_dir, return_metadata=True)
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_smithy/configure_feedstock.py", line 1221, in render_azure
    return_metadata=return_metadata,
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_smithy/configure_feedstock.py", line 608, in _render_ci_provider
    os.path.join(forge_dir, forge_config["recipe_dir"]), config=config
  File "/home/tobias/anaconda3/lib/python3.7/site-packages/conda_build/variants.py", line 521, in get_package_combined_spec
    raise ValueError("Error in config {}: {}".format(f, str(e)))
ValueError: Error in config /tmp/tmpa3nombf0/conda_build_config.yaml: Variant configuration errors: 
["zip_key entry fortran_compiler_version in group ['c_compiler_version', 'cxx_compiler_version', 'fortran_compiler_version'] does not have any settings"]
```

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

File: conda-forge.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
-build_platform: {osx_arm64: osx_64}
+build_platform: {osx_arm64: osx_64, linux_aarch64: default, linux_ppc64le: default}
```
Comment (isuruf) [inline conda-forge.yml:1]: You need `provider: linux_aarch64` instead of `build_platform: linux_aarch64`

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
Nice, thanks for that @isuruf. Rookie mistake :(

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please restart ci

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please restart ci

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please restart ci

**Comment by Tobias-Fischer** [comment]
I don't understand why it's still failing for aarch. Surely it can't be the CDN after 9 hours of meeting https://github.com/conda-forge/inotify_simple-feedstock/pull/15? Any clues @conda-forge/boa @conda-forge/help-osx-arm64?

**Comment by xhochy** [comment]
@Tobias-Fischer Rerun now and it seems it is green :)

**Comment by Tobias-Fischer** [comment]
Great, thanks @xhochy! This is ready to be merged @wolfv

**Pull request merged**